src/main/java/eu/smashmc/api/vote/VoteEntry.java

```diff
@@ -0,0 +1,43 @@
+package eu.smashmc.api.vote;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.net.InetAddress;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class VoteEntry {
+
+    /**
+     * Retrieve the player name, who voted
+     *
+     * @return player's name
+     */
+    private String playerName;
+
+    /**
+     * Get timestamp when the vote has been sent.
+     *
+     * @return timestamp
+     */
+    private String timestamp;
```

Comment:
The format is not clear. Please either change this to a java standard class (e.g. LocalDateTime) or tell the format in the documentation.

Reply:
Changed to long